coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -1365,11 +1365,14 @@ def test(self, backend, rank, dims):
         self.run_compare_torch(input_shape, model, backend=backend)
 
 class TestConstantPad(TorchBaseTest):
-    @pytest.mark.parametrize("backend", backends)
-    def test_constant_pad_1d(self, backend):
-        input_shape = (3, 4, 5)
-        model = torch.nn.ConstantPad1d((5, 6), 3.5).eval()
-        self.run_compare_torch(input_shape, model, backend=backend)
+    @pytest.mark.parametrize(
+        "input_shape, pad, backend",
+        itertools.product([2,2], (1,1), backends)
+    )
+    def test_constant_pad_nd(self, input_shape, pad, backend):
+        input_data = torch.rand(input_shape)
+        model = torch.constant_pad_nd(input_data, pad)
```

Comment:
There is an error in the CI:
```
    def test_constant_pad_nd(self, input_shape, pad, backend):
361        input_data = torch.rand(input_shape)
362>       model = torch.constant_pad_nd(input_data, pad)
363E       TypeError: constant_pad_nd(): argument 'pad' (position 2) must be tuple of ints, not int
```
I think the second argument in `itertools.product([2,2], (1,1), backends)` should be `[(1,1)]`.
Can you please check the test locally?

Reply:
Fixed it with my latest commit.